app/main.py

```diff
@@ -1,7 +1,25 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import requests
+import os
+
+
+def get_weather(api_key: str) -> None:
+    city = "Paris"
```

Comment:
Code Style Issue: The city name 'Paris' is hardcoded in the function. Consider making it a function argument or a constant at the top of your script, so it's easier to change in the future.